src/main/webapp/one-off/databaseInit.java

```diff
@@ -40,104 +39,123 @@
+      .addField(Field.newBuilder().setName("imgURL").setText(imgURL))
+      .build();
 
-    /**
-      * parseRecipe calls 'readFileInList' on a given text file, extrapolates all the recipes properties from the list of lines, and inputes them in 'upload'
-     **/
-    public void parseRecipe(String fileName) throws Exception, IOException  {
+    datastore.put(recipeEntity);
+    index.put(recipeDocument);
+  }
 
-       List<String> lines = readFileInList("INSERT ABSOLUTE PATH HERE"+fileName);
-       String title = lines.get(2);
-       String imgURL = lines.get(4);
-       int currentLineIndex = 7;
+  public static List<String> readFileInList(String fileName)
+    throws Exception, IOException {
+    List<String> lines = Collections.emptyList();
+    lines = Files.readAllLines(Paths.get(fileName), StandardCharsets.UTF_8);
+    return lines;
+  }
 
-       ArrayList<String> ingredients = new ArrayList<String>();
-       while (!lines.get(currentLineIndex).isBlank()){
-            ingredients.add(lines.get(currentLineIndex));
-            currentLineIndex += 1;
-       }
+  public void parseRecipe(String fileName) throws Exception, IOException {
+    List<String> lines = readFileInList(
+      "/home/donpaul/step/capstone/step249-2020/src/main/webapp/one-off/recipes/" +
+      fileName
+    );
+    String title = lines.get(2);
+    String imgURL = lines.get(4);
+    int currentLineIndex = 7;
 
-       currentLineIndex += 2;
-       ArrayList<String> steps = new ArrayList<String>();
-       while ((currentLineIndex < lines.size()) && (!lines.get(currentLineIndex).isBlank())){
-            steps.add(lines.get(currentLineIndex)); 
-            currentLineIndex += 1;
-       }
-       upload(title, imgURL, ingredients, steps);
-    
+    ArrayList<String> ingredients = new ArrayList<String>();
+    while (!StringUtils.isBlank(lines.get(currentLineIndex))) {
+      ingredients.add(lines.get(currentLineIndex));
+      currentLineIndex += 1;
     }
 
-   /**
-     * doGet loops trough all the 60 text files in the recipe folder an calls 'parseRecipe' on them
-    **/
-   @Override
-   public void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
-    
-    try{
+    currentLineIndex += 2;
+    ArrayList<String> steps = new ArrayList<String>();
+    while (
+      (currentLineIndex < lines.size()) &&
+      (!StringUtils.isBlank(lines.get(currentLineIndex)))
+    ) {
+      steps.add(lines.get(currentLineIndex));
+      currentLineIndex += 1;
+    }
+    upload(title, imgURL, ingredients, steps);
+  }
 
-     for ( int i = 1; i <= 60; i++) {
-       String nameFile = Integer.toString(i)+".txt";
-       parseRecipe(nameFile);
-     }
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+    throws ServletException, IOException {
+    try {
+      for (int i = 1; i <= 60; i++) {
+        String nameFile = Integer.toString(i) + ".txt";
```

Comment:
The TXT extension string should be placed in a constant

RECIPE_FILE_EXTENSION